src/certificates/certificate-signatories/CertificateSignatories.jsx

```diff
@@ -0,0 +1,117 @@
+import PropTypes from 'prop-types';
+import { Stack, Button, Form } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import CertificateSection from '../certificate-section/CertificateSection';
+import messages from './messages';
+import Signatory from './signatory/Signatory';
+import SignatoryForm from './signatory/SignatoryForm';
+import useEditSignatory from './hooks/useEditSignatory';
+import useCreateSignatory from './hooks/useCreateSignatory';
+
+const CertificateSignatories = ({
+  signatories, handleChange, handleBlur, arrayHelpers, setFieldValue, isForm, resetForm, editModes, setEditModes,
+}) => {
+  const intl = useIntl();
+
+  const {
+    toggleEditSignatory,
+    handleDeleteSignatory,
+    handleCancelUpdateSignatory,
+  } = useEditSignatory({
+    arrayHelpers, editModes, setEditModes, resetForm,
+  });
+
+  const { handleAddSignatory } = useCreateSignatory({ arrayHelpers });
+
+  return (
+    <CertificateSection
+      title={intl.formatMessage(messages.signatoriesSectionTitle)}
+      className="certificate-signatories"
+    >
+      <div>
+        <p className="mb-4.5">
+          {intl.formatMessage(messages.signatoriesRecommendation)}
+        </p>
+        <Stack gap="4.5">
```

Comment:
[Important]: The `gap` property only accepts a number as a value, now you are passing a string. Please correct it here and in other components of the Certificates page